ApiBundle/Transformer/ApiClientCollectionTransformer.php

```diff
@@ -2,7 +2,7 @@
 
 namespace OpenOrchestra\ApiBundle\Transformer;
 
-use Doctrine\Common\Collections\Collection;
+use Doctrine\Common\Collections\ArrayCollection;
```

Comment:
`ArrayCollection` extends `Collection`, but you could also get a PersistentCollection from the database.

I would prefer to let only the `Collection`, this allows more modification.